package.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "json-ld.org",
+  "private": true,
+  "description": "json-ld.org homepage",
+  "scripts": {
+    "build": "npx @11ty/eleventy",
+    "serve": "npx @11ty/eleventy --serve",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
```

This should probably be...
```suggestion
  "author": "Digital Bazaar, Inc.",
```
...or if we consider this a donation, then "W3C, Inc."

---

Removing the unused field.  This project uses package.json for build rules, not metadata.